### PR TITLE
docs(sentencizer): update readme and manifest

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
This pr is for JINAHUBV1-370. 
Let me know what else should we add to the content. 😄 

In the Manifest file:
- [x] short description at least includes the domain space 
- [x] the “documentation” points to the README (URL) of the respective Image
- [x]  “Keywords” include; audio, image, nlp, video

In the README:
- [x] markdown
- [x] 1-2 sentence description of the "Executor" and the functionality of it
- [x] Make sure we have a Use (Example Section) with code snipped of  "how to use it in flow"

### CODE REVIEW

**Comment by FionnD** [comment]
I'm adding @lusloher as a reviewer instead of me since this is her area :)

**Review by theUnkownName — DISMISSED** [review]

File: segmenters/nlp/Sentencizer/manifest.yml [hunk 1]
```diff
@@ -2,12 +2,12 @@ manifest_version: 1
...
   a rule-base strategy.
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
-documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.9
+documentation: https://github.com/jina-ai/jina-hub/blob/master/segmenters/nlp/Sentencizer/README.md
+version: 0.0.10
 license: apache-2.0
 keywords: [text, nlp, segement]
```
Comment (theUnkownName) [inline segmenters/nlp/Sentencizer/manifest.yml:13]: typo in Segment

File: segmenters/nlp/Sentencizer/README.md [hunk 2]
```diff
@@ -1,3 +1,53 @@
 # Sentencizer
 
-split the text on the doc-level into sentences on the chunk-level with a rule-base strategy. 
\ No newline at end of file
+Sentencizer is one segmenter used in nlp domain. It splits the text on the doc-level into sentences on the chunk-level with a rule-base strategy.
```
Comment (cristianmtr) [inline segmenters/nlp/Sentencizer/README.md:3]:
```suggestion
Sentencizer is one segmenter used in the NLP domain. It splits the text on the doc-level into sentences on the chunk-level with a rule-base strategy.
```

File: segmenters/nlp/Sentencizer/README.md [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+
+##Usage:
```
Comment (cristianmtr) [inline segmenters/nlp/Sentencizer/README.md:5]: Line break. Rendering broken

**Review by cristianmtr — CHANGES_REQUESTED** [review]
small thing

**Review by lusloher — APPROVED** [review]
Looks good from my side

**Review by FionnD — DISMISSED** [review]

File: segmenters/nlp/Sentencizer/manifest.yml (hunk 1 above)
Comment (Yongxuanzhang) [reply segmenters/nlp/Sentencizer/manifest.yml]: ✅

File: segmenters/nlp/Sentencizer/README.md (hunk 2 above)
Comment (Yongxuanzhang) [reply segmenters/nlp/Sentencizer/README.md]: ✅

File: segmenters/nlp/Sentencizer/README.md (hunk 3 above)
Comment (Yongxuanzhang) [reply segmenters/nlp/Sentencizer/README.md]: ✅

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**